src/rules/require-tothrow-message.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  ModifierName,
+  createRule,
+  isExpectCall,
+  parseExpectCall,
+} from './tsUtils';
+
+export default createRule({
+  name: __filename,
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description: 'Require a message for `toThrow()`',
+      recommended: false,
+    },
+    messages: {
+      requireRethrow: 'Add an error message to {{ propertyName }}()',
+    },
+    type: 'suggestion',
+    schema: [],
+  },
+  defaultOptions: [],
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (!isExpectCall(node)) {
+          return;
+        }
+
+        const { matcher, modifier } = parseExpectCall(node);
+
+        if (
+          matcher &&
+          matcher.arguments &&
+          matcher.arguments.length === 0 &&
+          ['toThrow', 'toThrowError'].includes(matcher.name) &&
+          (!modifier ||
+            !(modifier.name === ModifierName.not || modifier.negation))
```

Comment:
Why is both `not` and `negation` check needed?

Reply:
They're not, for the same reason `'negation' in matcher` isn't.

Reply:
>   ● require-tothrow-message › valid › expect(() => { throw new Error('a'); }).not.toThrowError();

That's why the `not` & negation is required.

`negation` is for `resolves` & `rejects`.